lib/main_common.pm

```diff
@@ -225,7 +225,7 @@ sub kdestep_is_applicable {
 }
 
 sub consolestep_is_applicable {
-    return !get_var("INSTALLONLY") && !get_var("DUALBOOT") && !get_var("RESCUECD");
+    return !get_var("INSTALLONLY") && !get_var("DUALBOOT") && !get_var("RESCUECD") && !check_var('FLAVOR', 'Gnome-Live');
```

Comment:
yeah, sure. Then maybe just ignore them for the kde-media as well? I don't see why we should make a difference for kde/gnome live media